coverage/sorter.js

```diff
@@ -161,7 +161,7 @@ var addSorting = (function() {
             return;
         }
         cols = loadColumns();
-        loadData(cols);
+        loadData();
```

Comment:
and this?

Reply:
Looped in w/  839c511. Thought you might have insight as to how changes to the `coverage` dir were automatically generated.